django/api/scheduled_jobs.py

```diff
@@ -12,6 +12,7 @@ def schedule_send_rebates_to_ncda():
             name="send_rebates_to_ncda",
             schedule_type="C",
             cron="15 * * * *",
+            q_options={"timeout": 1200, "ack_failure": True},
```

Comment:
Do we want to track failed tasks as successful using ack_failure: True? I could use a little background on the precedent with this one.

Reply:
For this particular job, I think it makes sense because, by default, ack_failure=False, and when that is the case, the task will keep on retrying until it succeeds and we want to avoid that with this job because of the ncda throttling issue.